Clarify monitoring read only role limitations (#1148)
rancher/rancher-docs: versioned_docs/version-2.7/integrations-in-rancher/monitoring-and-alerting/monitoring-and-alerting.md

## Role-based Access Control

For more information on configuring access to monitoring, see [this page.](rbac-for-monitoring.md)

:::note

> Rancher and Project read permissions do not necessarily apply to monitoring resources. See [monitoring-ui-view](rbac-for-monitoring.md#additional-monitoring-clusterroles) for more details.

Reviewer: ```suggestion
Rancher and Project read permissions don't necessarily apply to monitoring resources. See [monitoring-ui-view](rbac-for-monitoring.md#additional-monitoring-clusterroles) for more details.
```
